custom/languages/en_UK.json

```diff
@@ -911,6 +911,10 @@
     "installer/installer_upgrading_database": "Please wait whilst the installer upgrades your database...",
     "installer/installer_welcome": "Welcome to NamelessMC version 2.0",
     "installer/language": "Language",
+    "installer/module_core_description": "This module is required for NamelessMC to function correctly.",
+    "installer/module_cookie-consent_description": "Allows users to accept the use of cookies on your website.",
+    "installer/module_forum_description": "Adds a forum to your website.",
+    "installer/module_discord-integration_description": "Allows users to link their Discord account to their NamelessMC account and perform various synchronisation actions.",
```

Comment:
Perhaps "website account" might be clearer instead of "NamelessMC account" in case people think it's referring namelessmc.com? 😄

Reply:
Good point 😄